steps/bundle-universal-apk/0.0.1/step.yml

```diff
@@ -0,0 +1,80 @@
+title: Bundle Universal Apk
+summary: |
+  Generates an universal apk from an app bundle
+description: |
+  Runs a bundletool that gets an app bundle path and generates an universal apk that it can be upload to alternative app stores or test platforms.
+website: https://github.com/ch4vi/bitrise-step-bundle-universal-apk
+source_code_url: https://github.com/ch4vi/bitrise-step-bundle-universal-apk
+support_url: https://github.com/ch4vi/bitrise-step-bundle-universal-apk/issues
+published_at: 2019-01-24T12:48:04.825726+01:00
+source:
+  git: https://github.com/ch4vi/bitrise-step-bundle-universal-apk.git
+  commit: f6fb8fd939d9f04e24fc6ddd01b3b0655d944674
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- build
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
```

Comment:
curl and unzip are used in source but not declared as deps

Reply:
Sorry, but to be able to resolve those issues, Do I have to upload a new step version? Because it seems that I can't just commit the changes.
Thank you.